packages/mwp-app-render/src/util/seo/metaTags.test.js

```diff
@@ -0,0 +1,168 @@
+import locales from 'mwp-config/locales';
+import { MOCK_EVENT } from 'meetup-web-mocks/lib/api';
+
+import {
+	generateCanonicalUrl,
+} from './links';
+
+import {
+	generateMetaData,
+	generateGeoMetaData,
+} from './metaTags';
+
+const MOCK_META = {
+	title: 'mock title',
+	description: 'mock description',
+	keywords: 'mock keywords',
+	baseUrl: 'https://mock-base-url.com',
+	route: '/mock/preview/path',
+	appPath: '/mock/app/path',
+	imageUrl: 'https://www.meetup.com/mock-default-image.jpg',
+	localeCode: 'fr-FR',
+};
+
+describe('generateMetaData', () => {
+	const metaData = generateMetaData(MOCK_META);
+
+	it('should set the title', () => {
+		const titleMeta = metaData.filter(
+			obj => obj.content === MOCK_META.title && obj.property === 'og:title'
+		);
+		expect(titleMeta).toHaveLength(1);
+	});
+
+	it('should set the description', () => {
+		const descriptionMeta = metaData.filter(
+			obj =>
+				obj.content === MOCK_META.description &&
+				(obj.property === 'og:description' || obj.name === 'description')
+		);
+		expect(descriptionMeta).toHaveLength(2);
+	});
+
+	it('should set the keywords', () => {
+		const keywordsMeta = metaData.filter(
+			obj => obj.content === MOCK_META.keywords && obj.name === 'keywords'
+		);
+		expect(keywordsMeta).toHaveLength(1);
+	});
+
+	it('should set the app url', () => {
+		const appUrlMeta = metaData.filter(obj => obj.property === 'al:web:url');
+		expect(appUrlMeta).toHaveLength(1);
+	});
+
+	it('should set the appPath', () => {
+		const appPathMeta = metaData.filter(obj => obj.content === MOCK_META.appPath);
+		expect(appPathMeta).toHaveLength(2);
+	});
+
+	it('should set the url as localized url', () => {
+		const expectedUrl = generateCanonicalUrl(
+			MOCK_META.baseUrl,
+			MOCK_META.localeCode,
+			MOCK_META.route
+		);
+		const ogUrlMeta = metaData.filter(
+			obj => obj.content === expectedUrl && obj.property === 'og:url'
+		);
+		const alUrlMeta = metaData.filter(
+			obj => obj.content === expectedUrl && obj.property === 'al:web:url'
+		);
+		expect(ogUrlMeta).toHaveLength(1);
+		expect(alUrlMeta).toHaveLength(1);
+	});
+
+	describe('open graph meta data', () => {
+		it('should set og:site_name to "Meetup"', () => {
+			const ogSiteName = obj =>
```

Comment:
is this missing a `metaData.filter` call? the `expect(ogSiteName).toHaveLength(1);` is just checking the function, not the data.